can/io/blf.py

```diff
@@ -247,6 +284,8 @@ def __iter__(self):
                                       data=can_data[:dlc],
                                       channel=channel - 1)
                         yield msg
+                    # else:
+                    #     LOG.warning("Unknown object type (%d)", obj_type)
```

Comment:
could raise a exception/warning in this case

Reply:
That is exactly why I left the commented code in.

Open for discussion on if we should log a warning of not here.

Reply:
Okay, so: I think we should raise an Error and be yelled at the user/developer, since it probably means that he/she put in a file expecting it to be supported, but it is not.

Reply:
Log files contain lots of other information which should just be ignored. But it is good to have this commented out for debugging.